services/app-api/serverless.yml

```diff
@@ -144,6 +144,15 @@ functions:
           method: get
           cors: true
 
+  reports:
+    handler: src/handlers/reports.main
+    events:
+      - http:
+          path: reports
+          method: get
+          cors: true
+          authorizer: aws_iam
+
```

Comment:
Ah, you're gonna want to attach the Prisma layer for querying the DB -- also the OTEL layer for us to get telemetry data:
```
layers:
      - !Ref PrismaClientEngineLambdaLayer
      - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-2-0:1
```

Check out line 234 in the graphql handler for an example.